jena-arq/src/test/java/org/apache/jena/sparql/core/TestTestDatasetGraphMap.java

```diff
@@ -0,0 +1,9 @@
+package org.apache.jena.sparql.core;
+
+public class TestTestDatasetGraphMap extends AbstractDatasetGraphTests {
```

Comment:
"TestTest" -> "Test".

This also needs adding to `TS_Core` which it might a well go second in the list.

Reply:
Thanks for review, fixed.